lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorDescriptor.java

```diff
@@ -49,4 +52,9 @@
    * @return name of the runtime that was used to allocate this Evaluator
    */
   String getRuntimeName();
+
+  /**
+   * @return node label expressions allocated to this Evaluator
```

Comment:
Thank you for your comment!
Actually it means labels set on a node where the Evaluator is running.
I will update it.